akka-persistence-typed/src/test/java/akka/persistence/typed/javadsl/PersistentActorJavaDslTest.java

```diff
@@ -220,6 +220,17 @@ private Tick() {
     );
   }
 
+  private Behavior<Command> counterWithLoggingProbe(PersistenceId persistenceId, ActorRef<String> loggingProbe) {
```

Comment:
Looks very similar to the above `counter`. Can that `counter` delegate to this one?